backend/core/utils/tests/test_time.py

```diff
@@ -8,3 +8,12 @@ def test_time_ago():
     assert time_ago(hours=10) > time_ago(days=1)
     assert time_ago(days=1) > time_ago(days=1, hours= 3)
     assert time_ago(days=6, hours=23) > time_ago(weeks=1)
+
+
+def test_time_ahead():
```

Comment:
Out of scope: in the future, if we want to make those tests more robust, we could freeze the time and compare the results of `time_ago` and `time_ahead` to now. Currently those tests don't check that the returned times are relative to now, which is a key part of the implementation of `time_ago` and `time_ahead. 

For instance we could check that `time_ahaed(hours=1)` returns a `datetime` object 3600 seconds in the future.

Reply:
Yes I thought the same, but I wasn't sure it it was overkill to test that

Reply:
The functions are really small indeed, but it will be quite easy to test their relativity to now. In my opinion it's not overkill. Is it worth the invested time? That's also a question : ]